ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -165,6 +167,22 @@ instance (ByronGiven, Typeable cfg) => Measured BlockMeasure (ByronBlock cfg) wh
 
 instance StandardHash (ByronBlock cfg)
 
+{-------------------------------------------------------------------------------
+  Secrets
+-------------------------------------------------------------------------------}
+
+-- This is specifically including Pbft in the name, because Byron Classic
+-- also required a VSS keypair, which Pbft does not.
+--
+-- Alternatively, this type could have been placed in the PBFT protocol,
+-- under a different name, but it would have been even more unappropriately
+-- over-specific there.
```

Comment:
I'm not sure what to make of this comment. It's not API documentation. It seems to really be a commit message or PR review message suggesting alternative ways of doing it.

Reply:
Perhaps this type only needs to be defined in `Ouroboros.Consensus.Node.ProtocolInfo.Byron`

As for naming, perhaps `PbftLeaderCredentials`? Yes, mentioning PPBT makes sense. The inclding "byron" in the name is unnecessary, as PBFT is only for that era.

@nc6 opinions on either/both?

Reply:
@nc6, waiting for your take!

Meanwhile I've implemented the initial Duncan's suggestion:

1. renamed to `PbftLeaderCredentials`
2. moved to `Ouroboros.Consensus.Node.ProtocolInfo.Byron`